Dockerfile

```diff
@@ -1,42 +1,32 @@
 ARG ONOS_BUILD_VERSION=stable
-FROM debian:stretch
+FROM golang:1.11-alpine
```

Comment:
should be fine for simulator

Reply:
Since we need golang, it is recommended to use golang alpine image as the base image since package manager in alpine is different than Debian and we need to add lots of things to the Dockerfile to install golang.